<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnalogGaugeWidget_Demo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>651</width>
         <height>731</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_28">
          <property name="font">
           <font>
            <pointsize>30</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ITI:NC|G1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="SpeedLCD"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="AnalogGaugeWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>600</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_32">
          <property name="font">
           <font>
            <pointsize>30</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SPEED :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="ButtonDistance50">
       <property name="geometry">
        <rect>
         <x>1250</x>
         <y>80</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Distance:50CM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonDistance75">
       <property name="geometry">
        <rect>
         <x>1250</x>
         <y>150</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Distance:75CM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonDistance100">
       <property name="geometry">
        <rect>
         <x>1250</x>
         <y>220</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Distance:100CM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonACC_ON">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>400</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ACC ON</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonCC_OFF">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>560</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CC OFF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonNCC_ON">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>480</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>NCC ON</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="TextCC_O_Mode">
       <property name="geometry">
        <rect>
         <x>1060</x>
         <y>430</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="TextOCC_Statue">
       <property name="geometry">
        <rect>
         <x>1060</x>
         <y>510</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="TextODistanceMode">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>120</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>100</y>
         <width>321</width>
         <height>95</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current mode : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>410</y>
         <width>351</width>
         <height>95</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current CC mode :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>500</y>
         <width>361</width>
         <height>95</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current CC statue : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>1230</x>
         <y>30</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Distance Modes</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>1280</x>
         <y>340</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CC Modes</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogGaugeWidget</class>
   <extends>QWidget</extends>
   <header location="global">analoggaugewidget</header>
   <container>1</container>
   <slots>
    <signal>valueChanged(int)</signal>
    <slot>update_angle_offset(int)</slot>
    <slot>update_value(int)</slot>
    <slot>update_value2(int)</slot>
    <slot>set_start_scale_angle(int)</slot>
    <slot>set_total_scale_angle_size(int)</slot>
    <slot>set_MinValue(int)</slot>
    <slot>set_MaxValue(int)</slot>
    <slot>set_NeedleColor(int)</slot>
    <slot>set_scala_main_count(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
